haste.py

```diff
@@ -27,49 +25,73 @@ async def handler(event):
             return
 
         msg = await event.get_reply_message()
-        if len(msg.raw_text or '') < 200:
-            sent = await event.respond('Not bothering to paste such a short message.')
+
+        formatter_url = "https://1rctyledh3.execute-api.us-east-1.amazonaws.com/dev"
+        payload = {"source": msg.raw_text}
+        headers = {"content-type": "application/json"}
+
+        if len(msg.raw_text or "") < 200:
+            sent = await event.respond("Not bothering to paste such a short message.")
             await asyncio.sleep(10)
             await sent.delete()
             return
 
         if event.id - msg.id > _MAX_MSG_DISTANCE:
-            sent = await event.respond('The message is too old for a paste to matter now.')
+            sent = await event.respond(
+                "The message is too old for a paste to matter now."
+            )
```

Comment:
There's no need to be so strict with the column limit, it just hurts readability.